### PR TITLE
fix platform issue

### PR DESCRIPTION
**Pull request by jiaqiluo**
fix platform issue

### CODE REVIEW

**Review by kinarashah — APPROVED** [review]

**Review by HarrisonWAffel — APPROVED** [review]

File: libmachine/drivers/plugin/localbinary/utils_linux.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package localbinary
```
Comment (HarrisonWAffel) [inline libmachine/drivers/plugin/localbinary/utils_linux.go:1]: This file also needs build tags 

```
//go:build !windows
// +build !windows
```

File: libmachine/drivers/plugin/localbinary/utils_nolinux.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
+//go:build !linux
```
Comment (HarrisonWAffel) [inline libmachine/drivers/plugin/localbinary/utils_nolinux.go:1]: These should be 
```
//go:build windows
// +build windows
```
to match how other repositories handle multiple platforms

**Review by HarrisonWAffel — CHANGES_REQUESTED** [review]

File: libmachine/drivers/plugin/localbinary/utils_nolinux.go (hunk 2 above)
Comment (HarrisonWAffel) [reply libmachine/drivers/plugin/localbinary/utils_nolinux.go:1]: Also, the file should have a suffix `_windows.go`, to match how other repositories handle this case.

File: libmachine/drivers/plugin/localbinary/utils_linux.go (hunk 1 above)
Comment (jiaqiluo) [reply libmachine/drivers/plugin/localbinary/utils_linux.go]: Fixed.  

I also removed the `+build` tag which is the older way of doing conditional compilation before GO 1.17

File: libmachine/drivers/plugin/localbinary/utils_nolinux.go (hunk 2 above)
Comment (jiaqiluo) [reply libmachine/drivers/plugin/localbinary/utils_nolinux.go]: fixed. 

The other file is renamed to `utils.go` as it is for every platform except windows

**Review by HarrisonWAffel — APPROVED** [review]

**Review by kinarashah — APPROVED** [review]

**Pull request merged**